inventory_management_system_api/schemas/item.py

```diff
@@ -27,7 +27,7 @@ class ItemPostRequestSchema(BaseModel):
     """
 
     catalogue_item_id: str = Field(description="The ID of the corresponding catalogue item for this item")
-    system_id: Optional[str] = Field(default=None, description="The ID of the system that the item belongs to")
+    system_id: str = Field(default=None, description="The ID of the system that the item belongs to")
```

Comment:
Can `default=None` be removed now?
```suggestion
    system_id: str = Field(description="The ID of the system that the item belongs to")
```